### ISSUE TITLE
Phoenix Wright: Ace Attorney Trilogy - CUSA11520

### ISSUE THREAD

**Issue by Addy2004** [red-prig/fpps4-game-compatibility#681]
# Game name
Phoenix Wright: Ace Attorney Trilogy

## Code
CUSA11520

## Used emulator's version
red-prig/fpPS4@dbfab84

## Current status
Boots

## Error
```
no error prompted, couldn't find one in log
``` 

## Description
Boots with a black screen.

## Screenshots
![image](https://user-images.githubusercontent.com/40354792/209447861-27fa5f48-d6f8-4c5c-9093-21828cd95379.png)

## Log
[Log_Phoenix_Wright_Ace_Attorney_Trilogy_Sun_Dec_25_2022_0_7_43.log](https://github.com/red-prig/fpps4-game-compatibility/files/10299033/Log_Phoenix_Wright_Ace_Attorney_Trilogy_Sun_Dec_25_2022_0_7_43.log)

**Comment by gandalfthewhite19890404** [comment]
Try hacks, especially COMPUTE_DISABLE_HACK

**Comment by Addy2004** [comment]
> Try hacks, especially COMPUTE_DISABLE_HACK

Yeah, I tried all of them, individually and in combination but couldn't get any progress or error. You can try this game if you wish to, it's small and not that much demanding.

**Comment by Kalorte** [comment]
I can find this error in red:

## Error
vkQueueSubmit:VK_ERROR_DEVICE_LOST
vkQueueSubmit:VK_ERROR_DEVICE_LOST
vkQueuePresentKHR:VK_ERROR_DEVICE_LOST
[PlatformThread:21544] BeginThread:0000000800649F00
[PlatformThread:21544] EndThread:PlatformThread
vkQueueSubmit:VK_ERROR_DEVICE_LOST

## Emulator version
https://github.com/red-prig/fpPS4/commit/41b20dcda94bbffe7677470d308dc39758ce5a60

**Comment by Addy2004** [comment]
Can you please attach full log here?

**Comment by Kalorte** [comment]
[phoenix.log](https://github.com/red-prig/fpps4-game-compatibility/files/10367094/phoenix.log)

**Comment by Addy2004** [comment]
> [phoenix.log](https://github.com/red-prig/fpps4-game-compatibility/files/10367094/phoenix.log)
I read your log, it seem like it goes further for you than in my testing. For the `VK_ERROR_DEVICE_LOST`, I would recommend using `MEMORY_BOUND_HACKS` or `COMPUTE_DISABLE_HACK` as it might help.
